page.go

```diff
@@ -464,6 +464,126 @@ func (p *Page) Screenshot(fullPage bool, req *proto.PageCaptureScreenshot) ([]by
 	return shot.Data, nil
 }
 
+type ScrollScreenshotOptions struct {
+	// Format (optional) Image compression format (defaults to png).
+	Format proto.PageCaptureScreenshotFormat `json:"format,omitempty"`
+
+	// Quality (optional) Compression quality from range [0..100] (jpeg only).
+	Quality *int `json:"quality,omitempty"`
+
+	// FixedTop (optional) The number of pixels to skip from the top.
+	// It is suitable for optimizing the screenshot effect when there is a fixed positioning element at the top of the page.
+	FixedTop float64
+
+	// FixedBottom (optional) The number of pixels to skip from the bottom.
+	FixedBottom float64
+
+	// WaitPreScroll wait scroll animation (default is 500ms)
+	WaitPreScroll time.Duration
+}
+
+// ScrollScreenshot Scroll screenshot does not adjust the size of the viewport, but achieves it by scrolling and capturing screenshots in a loop, and then stitching them together.
+// Note that this method also has a flaw: when there are elements with fixed positioning on the page (usually header navigation components), these elements will appear repeatedly,
+//
+//	you can set the FixedTop parameter to optimize it.
+func (p *Page) ScrollScreenshot(opt *ScrollScreenshotOptions) ([]byte, error) {
```

Comment:
Could you add a test case to `page_test.go`, or the CI will fail, we require 100% code coverage.

Check this doc: https://github.com/go-rod/rod/blob/main/.github/CONTRIBUTING.md#run-tests